sdk/core/core-http/lib/axiosHttpClient.ts

```diff
@@ -125,7 +125,7 @@ export class AxiosHttpClient implements HttpClient {
         url: httpRequest.url,
         headers: rawHeaders,
         data: axiosBody,
-        transformResponse: undefined,
+        transformResponse: (data) => { return data; },
```

Comment:
This change is due to us pulling changes from ms-rest-js and not related to the identity work, right?

Reply:
Yep, this is a fix for a bug that was introduced in a very recent change to `ms-rest-js` which got pulled in when we did the `@azure/core-http` migration.